packages/runtime/datastore/src/channelStorageService.ts

```diff
@@ -55,7 +55,13 @@ export class ChannelStorageService implements IChannelStorageService {
             ? this.extraBlobs.get(id)
             : undefined;
 
-        return blob !== undefined ? stringToBuffer(blob,"base64") : this.storage.readBlob(id);
+        if (blob !== undefined) {
+            return stringToBuffer(blob, "base64");
+        }
+
+        // for back compat. will be replaced with readBlob after version 0.35
+        const res = await this.storage.read(id);
+        return stringToBuffer(res, "base64");
```

Comment:
if we use this.storage.readBlob here now, the back compat tests will fail, because it will call the old readBlob which does not read from cache like read. So I'm only keeping this as read here. If I can debug and find the solution before version 0.35 is ready, I will change it back. Otherwise I will wait for 0.35.

Reply:
Sounds good - please open an issue not to forget

Reply:
@vladsud